package.json

```diff
@@ -23,16 +23,19 @@
     "jest-circus": "^26.6.3",
     "jscodeshift": "^0.11.0",
     "kolorist": "^1.2.10",
+    "preact": "^10.5.12",
     "prettier": "^2.2.1",
     "rollup": "^2.36.1",
     "rollup-plugin-prettier": "^2.1.0",
     "rollup-plugin-terser": "^7.0.2",
+    "sass": "^1.32.6",
     "smoldash": "^0.9.0",
+    "source-map": "^0.7.3",
     "typescript": "^4.1.3"
   },
   "dependencies": {
     "@types/testing-library__jest-dom": "^5.9.5",
-    "puppeteer": "^5.5.0",
+    "puppeteer": "^6.0.0",
```

Comment:
They released this today and it uses the first Chromium version that supports top-level await, which is convenient because that's just what I needed [here](https://github.com/cloudfour/test-mule/pull/17/files#diff-91fbb1d3b9ae2a143e68e5008ce5697d5fb6a4979da129fd57c8c219d90d66ceR55)